### PR TITLE
WhoIsWatching additional config

### PR DESCRIPTION
**Pull request by Amanda1234567**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -5545,6 +5545,26 @@
 		'wisdomwikiwiki' => 'freenodeChat',
 		'wisdomsandboxwiki' => 'freenodeChat',
 	),
+	
+	//WhoIsWatching
+	$whoiswatching_nametype => array(
+		'default' => "RealName";
```
Comment (The-Voidwalker) [inline LocalSettings.php]: you need to replace all semicolons `;` with commas.

**Comment by Amanda1234567** [comment]
@The-Voidwalker done

**Comment by Amanda1234567** [comment]
@JohnFLewis @Reception123 please merge as soon as possible

File: LocalSettings.php [hunk 2]
```diff
@@ -5545,6 +5545,26 @@
 		'wisdomwikiwiki' => 'freenodeChat',
 		'wisdomsandboxwiki' => 'freenodeChat',
 	),
+	
+	//WhoIsWatching
+	$whoiswatching_nametype => array(
```
Comment (Reception123) [inline LocalSettings.php]: Please do it in the format that all the other variables are done which requires a ', for example '$wgWhoIsWatching_Nametype', etc.

Also the "= array" is supposed to be a "=> array"

File: LocalSettings.php [hunk 3]
```diff
@@ -5545,6 +5545,26 @@
 		'wisdomwikiwiki' => 'freenodeChat',
 		'wisdomsandboxwiki' => 'freenodeChat',
 	),
+	
+	//WhoIsWatching
```
Comment (Reception123) [inline LocalSettings.php]: Minor thing but a space should be added so its // WhoIsWatching

**Comment by Amanda1234567** [comment]
@Reception123 both done

**Comment by Reception123** [comment]
I'm not sure if the $ is still required or not. We will have to wait for someone (most likely John) to confirm this.

**Comment by JohnFLewis** [comment]
@reception123 it looks fine

**Pull request merged**